android/app/src/main/java/app/organicmaps/bookmarks/Holders.java

```diff
@@ -189,14 +189,12 @@ private String getSizeString()
       final int bookmarksCount = mEntity.getBookmarksCount();
       final int tracksCount = mEntity.getTracksCount();
 
-      if (mEntity.size() == 0)
-        return getQuantified(resources, R.plurals.objects, 0);
-
-      if (bookmarksCount > 0 && tracksCount > 0)
+      if (bookmarksCount == 0 && tracksCount == 0 || bookmarksCount > 0 && tracksCount > 0)
       {
         final String bookmarks = getQuantified(resources, R.plurals.places, bookmarksCount);
```

Comment:
iOS uses `bookmarks_places` string, it would be better to use it on iOS too, and delete the `places` string.